src/bank02_player_menus.asm

```diff
@@ -6884,16 +6916,19 @@ call_02_75f4:
     ld   H, A                                          ;; 02:760d $67
     ld   A, [wMoneyLow]                                ;; 02:760e $fa $be $d7
     ld   L, A                                          ;; 02:7611 $6f
```

Comment:
This and the line above it should be able to be changed to:
`ld hl, wMoneyLow
ld a, [hl+]
ld h, [hl]
ld l, a`

That would be a two byte savings.

Reply:
Done